app/assets/javascripts/application.js

```diff
@@ -1,4 +1,5 @@
 //= require jquery/dist/jquery.min.js
 //= require click-tracker.js
+//= require take-whole-pot-calculator
```

Comment:
The JS version doesn't have validation. The plan was to get this out and then let @benbarnett loose on it.